blog/2023-12-22-openhab-4-1-release.md

```diff
@@ -59,6 +59,33 @@ This can be used for example to add a location or equipment that does not (yet)
 To define your own semantic tags, create a YAML configuration file containing the description of all your specific semantic tags in the `$OPENHAB_CONF/tags` folder.
 You can find a [tutorial on the community forum](https://community.openhab.org/t/oh-4-1-tutorial-to-manage-custom-semantic-tags/148135).
 
+### Voice System Enhancements
+
+_Miguel Álvarez Díez, openHAB Contributor_
+
+Some minor improvements were introduced in the built-in voice system components. 
+
+The standard text interpreter now understand your Item synonyms, 
+and gives priority to an exact match on the label/synonym allowing an easier control over the command target.
+It also understand your Item metadata descriptions, 
+the rule `Set <ItemLabel> to <ItemCommandLabel>` is available out-of-the-box for the supported languages for all the Items with the commandDescription metadata.
+
+In the same direction the `voiceSystem` metadata namespace was created to allow adding custom rules of your need, for example:
+
+* You can add the rule `go to mode $cmd$ on $name$` to an Item labeled `Vacuum` with command description `MODE_CLEAN=cleaning`,
+that will instruct the interpreter to send the command `MODE_CLEAN` to the Item on the phrase `go to mode cleaning on vacuum`.
+
+* You can also capture free text with a rule like `watch $*$ on $name$`,
+the phrase `watch my favorite show on tv` will send the command `my favorite show` to the Item labeled as `TV`.
+
+Two new configuration parameters where added to the dialog processor start/register command:
+
+* The `dialogGroup` option prevents simultaneous execution for dialogs on the same group,
+so you can have near speakers and be sure those won't be triggered at the same time.
```

Comment:
What about:

> The `dialogGroup` option prevents simultaneous processing of dialogs on the same group, so that several speakers located near-by won't be triggered at the same time.